### PR TITLE
PrebuiltModule: generate ABI baseline files as byproducts of generating prebuilt modules

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please test

**Comment by nkcsgexi** [comment]
@swift-ci please test

**Comment by nkcsgexi** [comment]
@swift-ci please test

File: Sources/SwiftDriver/Jobs/PrebuiltModulesJob.swift [hunk 1]
```diff
@@ -357,21 +455,46 @@ extension Driver {
     commandLine.appendFlag(.serializeParseableModuleInterfaceDependencyHashes)
     commandLine.appendFlag(.badFileDescriptorRetryCount)
     commandLine.appendFlag("30")
-    return Job(
+    var allOutputs: [TypedVirtualPath] = [outputPath.path]
+    var allJobs: [Job] = []
+    // Emit ABI descriptor if the output dir is given
+    if let currentABIDir = currentABIDir, isFrontendArgSupported(.emitAbiDescriptorPath) {
+      commandLine.appendFlag(.emitAbiDescriptorPath)
+      // Derive ABI descritor path from the prebuilt module path.
```
Comment (artemcm) [inline Sources/SwiftDriver/Jobs/PrebuiltModulesJob.swift:463]:
```suggestion
      // Derive ABI descriptor path from the prebuilt module path.
```

**Review by artemcm — APPROVED** [review]
This looks great!

**Pull request merged**